### PR TITLE
Updated Salesforce integration

### PR DESCRIPTION
**Pull request by yaakovi**
fixes: https://github.com/demisto/etc/issues/6766

### CODE REVIEW

**Comment by yaakovi** [comment]
documentation for the integration:
[Salesforce.docx](https://github.com/demisto/content/files/1785095/Salesforce.docx)

@mayairv @michalgold

**Comment by mayairv** [comment]
@yaakovi 
The following article has been added please review and approve for external publication https://support.demisto.com/hc/en-us/articles/360001848133-Integration-Salesforce

**Comment by mayairv** [comment]
The article was published here: https://support.demisto.com/hc/en-us/articles/360001848133-Integration-Salesforce

File: Integrations/integration-Salesforce.yml [hunk 1]
```diff
@@ -4,9 +4,20 @@ commonfields:
...
...8oDDw0mZO5k1Rwdk3f9ZtnfkYyi/kaAZAwAAIB8vkYrDl0q604XN4DGlMs6XjJ7y+wT5fkdiRWOS6t7cURh6OYpnaeYyAAAwXEnQ7PNkzf4sYcGWdEFppXWKtm630pnO1PBH5FL9GuH6kXPQbAMAAJDTIirq9hWU+T+hVnQwxw8Q+aZwbaEveU00w5g38VDTP4SZ+xOpVPswyg/IaC4BAABsCBeLjddizf0SscJHKA/QSeBKr5lpzoLbvq0wL/33EdPnczV+KhXtL4pq/TVzNzgDAABogzRbGnq7qNk9xAxadE0SpO8Y5weI3Hm/2JUmr7nk30OT3xJIpR1iw+nCpdG/QPl1BAEAAMyHA0RUGHiRrI5+AJvhHdQMfpW7a2ec69s6MQdZd+ZlYFb62rFy544YG/5uiXl9WHPeK/YMnY0GoG0bAID5RmFdhmr15RJzP4nN4B7FCncra3fE/DBxXgxe0uyJO81JeztNvm7C/D2y6d+TVe2PZgr15ajf7kAAAADzmvu3i0SNLiTlxocwcy3ZcPfm7tzOSwWTNWp7DgVrO6b8azH9uHPt9lhZuzPGhrdLMNw75OTrz/Ruugj1bxAQgutzAAAsGPKnoU8MZIVi/VyhVFtBNceUteYO0jce53hWXRmLFWM2yx9O+sMit3prrKzZEVM94FfmBjFr3pgx7Heh3vXno3VBZsHPbwYAAG58LOt5+Gyh59E3CcXGl2Q9+G9i+Lt4KUFZ+3isVFtpBjvdm1wo8xL9ib8vWUpAk1dZdz1R9wy5bH9Q7t18CTLsTrQyXooAAABOta0tqLhL7uRZdWHL65eqjatlvakSrWkT3d2LmZ/Wf3kppNOKeIY7BQPzo+T/ty1W+vgVuSAmWnOvrDUa/NZJGpTLo69At4yega49mi0DAABAVyLPrBOJuNo5S1g18maxZ/SfiGaXJKP5I5m5Taw5T1DdPcIDLZcyf1L+3j1GP3XycxPZuO4eJqqzG+vNBladB4SSXRDV+jU4+XuE8sg56PsuTg/9eCnmjwEAAACsWIy0eClaF2eQ6WKUZNlo1aOvWNK96a1LCyN/ny3bN4rluimWGv8pq86DiY/Iqr1FKDXWZ4v1HyWv382qrtWh2XmBd/dZm9/Y0bPhlei2zeei/gEZ5QeyaOXg0gW5dgoAAAAAAOD/AJTdWvpkIYlkAAAAAElFTkSuQmCC
```
Comment (asafshen) [inline Integrations/integration-Salesforce.yml]: @yaakovi did you get the image from Eynav? it seems larger. it should be around 4KB

Comment (yaakovi) [reply Integrations/integration-Salesforce.yml]: no, i only improved the integration so the image was already there

**Pull request merged**